hoodie-client/src/main/java/com/uber/hoodie/table/HoodieCopyOnWriteTable.java

```diff
@@ -575,6 +577,98 @@ protected HoodieMergeHandle getUpdateHandle(String commitTime, String fileLoc,
     return stats;
   }
 
+  @Override
+  public void initializeFinalizeWrite() {
+    if (!config.shouldUseTempFolderForCopyOnWrite()) {
```

Comment:
Whats the cost of always creating a `.hoodie/tmp` folder? or during table init based on config.. 

Adding a new method to table for this seems like an overkill.